zerver/lib/send_email.py

```diff
@@ -39,11 +39,13 @@ def build_email(template_prefix, to_email, from_name=None, from_address=None,
     if from_address is None:
         from_address = FromAddress.NOREPLY
     from_email = formataddr((from_name, from_address))
-    reply_to = None
+
     if reply_to_email is not None:
-        reply_to = [reply_to_email]
+        reply_to = reply_to_email
+    else:
+        reply_to = from_email
```

Comment:
I am not confident, but it seems like potentially a violation of how emails are normally generated to set `reply_to` equal to the from email address automatically.  I don't think symmetry is a good reason to do this; why not just leave it as None here?